src/backend/distributed/shared_library_init.c

```diff
@@ -1262,6 +1262,17 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL | GUC_NOT_IN_SAMPLE,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.enable_create_database_propagation",
+		gettext_noop("Enables propagating CREATE DATABASE "
+					 "and DROP DATABASE statements to workers"),
+		NULL,
+		&EnableCreateDatabasePropagation,
+		false,
```

Comment:
Let's make this `true` by default.

Reply:
When I change it to true, I need to change tests up to 10 to make it work since all these tests are working based on non-propagated database create and drop statements. I think I need to change it after merging this PR since it will create a high level of complexity for this PR to be approved

Reply:
Alright, let's leave it false by default for now. Especially because we don't support some of the options, like template and collation related ones.